pkg/network/bridge/plugin.go

```diff
@@ -114,27 +122,40 @@ func (nbdp *NetworkBridgeDevicePlugin) Allocate(ctx context.Context, r *pluginap
 
 	for _, req := range r.ContainerRequests {
 		var devices []*pluginapi.DeviceSpec
+		var vnics []vnic
 		for _, nic := range req.DevicesIDs {
 			dev := new(pluginapi.DeviceSpec)
 			assignmentPath := getAssignmentPath(nbdp.bridge, nic)
 			dev.HostPath = fakeDevicePath
 			dev.ContainerPath = assignmentPath
 			dev.Permissions = "r"
 			devices = append(devices, dev)
+			vnics = append(vnics, vnic{nic, protocolEthernet})
 
 			nbdp.assignmentCh <- &Assignment{
 				nic,
 				assignmentPath,
 				time.Now(),
 			}
 		}
+		envs := map[string]string{
+			envVariableName: marshal(nbdp.bridge, vnics),
+		}
 		response.ContainerResponses = append(response.ContainerResponses, &pluginapi.ContainerAllocateResponse{
-			Devices: devices})
+			Devices: devices, Envs: envs,
+		})
+
 	}
 
 	return &response, nil
 }
 
+func marshal(interfaceName string, vnics []vnic) string {
+	vnicsPerInterface := map[string][]vnic{interfaceName: vnics}
+	marshalled, _ := json.Marshal(vnicsPerInterface)
```

Comment:
what would be the format inside the env variable? Seems like I expect something different.
On my end I expect, for example:
```
{"name": "network-bridge.dp.io/red", "interfaces": [{"name": "eth-red01", "protocol": "Ethernet"},{"name": "eth-red02", "protocol": "Ethernet"}]}
```

Reply:
Done. Previously I tried to match
NETWORK_INTERFACE_RESOURCES_<unique identified for your plugin>=’{“mybr0”: [{“name”: “mybr0-01”, “protocol”: “IP|Ethernet|PCI”, {"name":...}]}’

Reply:
Please note that this works currently only for a single bridge interface / vNIC